src/handlers/sendsqs.js

```diff
@@ -0,0 +1,39 @@
+// Load the AWS SDK for Node.js
+const AWS = require('aws-sdk');
+const logger = require('../utils/logger')
+// Set the region 
+AWS.config.update({region: 'eu-west-2'});
+
+
+const sendMessage = async (event) => {
+  // Create an SQS service object
+  const sqs = new AWS.SQS();
+
+  const queueUrl = 'https://sqs.eu-west-2.amazonaws.com/533322355632/myfirstsqsqueue'
```

Comment:
```suggestion
  const queueUrl = process.env.QUEUE_URL
```
You could stick this in the environment of the lambda